modules/daicard/src/components/sendCard.js

```diff
@@ -123,6 +123,7 @@ export const SendCard = style(
       }
       paymentAction("DONE");
     };
+    window.paymentHandler = paymentHandler;
```

Comment:
This allows me to instrument the UI if I want to. It will be removed prior to merge.